### ISSUE TITLE
Web Interface Should Register a Protocol Handler for RSS in Firefox

### ISSUE THREAD

**Issue by barberousse** [feedbin/support#412]
Seems like a nice-to-have. Makes it easier to feverishly add subscriptions.. See the [MDN page](https://developer.mozilla.org/en-US/docs/Web-based_protocol_handlers) for how-to.

**Comment by rdrey** [comment]
Ah, I see. I had to manually run the JS because I created my account in Chrome, but then switched to Firefox.

**Comment by OJFord** [comment]
I agree, there are so many circumstances through which a user can 'miss' the opportunity to register - not least "I'm just trying it out, I'll do that later if I like it", which seems a fairly likely reason to dismiss it initially, and then never be shown it again.

**Suggestion**: somewhere in settings, have an option to 'add to FF as feed reader', which just runs this on click. If it's already working, a user won't click it and won't have the 'already registered' pop-up. Perhaps have it under 'Import/Export'.
